src/types/AdviceData.ts

```diff
@@ -0,0 +1,6 @@
+export type AdviceData = {
+  slip: {
+    id: number;
+    advice: string;
+  };
+}
```

Comment:
Tal vez el nombre del type podría ser algo como: AdviceResponseBody y hacer referencia del Advice en la propiedad slip:

```suggestion
export type AdviceResponseBody = {
  slip: Advice
}
```

Ese type podría estar en el Advice.ts